ses/main.tf

```diff
@@ -10,23 +10,23 @@ resource "aws_route53_record" "amazonses_verification_record" {
   records = [aws_ses_domain_identity.email_domain.verification_token]
 }
 
-# include environment in conditional statement until DNS delegation is in place
+# implement this resource after DNS delegation is in place to avoid terraform errors
 resource "aws_ses_domain_identity_verification" "email_verification" {
-  count  = var.environment_full_name == "production" ? 0 : 1
+  count  = var.domain == true ? 0 : 1
   domain = aws_ses_domain_identity.email_domain.id
```

Comment:
The domain variable is a string set to nationalarchives.gov.uk as far as I can see, should we compare it to `true` like this

Reply:
thanks, corrected